soh/soh/Enhancements/randomizer/3drando/item_pool.cpp

```diff
@@ -662,9 +662,17 @@ void GenerateItemPool() {
     IceTrapModels.push_back(0xD3);
   }
 
+  if (TriforceHunt.Is(TRIFORCE_HUNT_ON)) {
+    IceTrapModels.push_back(0xDF);
```

Comment:
unfortunate that these are magic numbers, but it seems that's not a new thing and cleaning them all up here would bloat this PR

Reply:
Agreed, I'm sorta hoping this is one of those things we might be able to tackle during a randomizer rework.